django_freeradius/settings.py

```diff
@@ -35,3 +35,5 @@
                                 os.path.dirname(__file__),
                                 'templates/django_freeradius/prefix_pdf.html')
                              )
+
+API_TOKEN = getattr(settings, 'DJANGO_FREERADIUS_API_TOKEN', None)
```

Comment:
oh sorry. I just realized this! What happens if the app is initialized with no token?
I think it the safest thing is to add a `ready` method here: https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/apps.py
and raise `ImproperlyConfigured` if the token is not a string of at least 15 characters long

Reply:
Fixed.